pom.xml

```diff
@@ -203,8 +203,9 @@
 
         <curator.version>2.12.0</curator.version>
 
-        <avro.version>1.11.0</avro.version>
+        <avro.version>1.10.1</avro.version>
```

Comment:
Why downgrade this version?

Reply:
in org.apache.iceberg.avro.Avro`, here it's the code.
![image](https://user-images.githubusercontent.com/37214832/168546861-f4259d8b-e771-4879-b22c-5f028259c897.png)
`org.apache.avro.LogicalTypes#register` implement different in 1.10.1 and 1.11.0.
In 1.11.0，it will get Type Name，
![image](https://user-images.githubusercontent.com/37214832/168547622-1cae1420-3a32-4725-863a-b222fcf85f77.png)

We can't solve this problem unless we modify iceberg code.

Reply:
Okay, thanks for your reply.